ramalama/console.py

```diff
@@ -0,0 +1,39 @@
+import os
+import logging
+
+
+def is_locale_utf8():
```

Comment:
I tested this on macOS with alacritty it needs one more check to work, because this is the only of the 3 env vars populated on this platform:

```
 $ echo $LC_ALL
 en_IE.UTF-8
 ```
 
 So we need to add one more or:
 
 ```
 return 'UTF-8' in os.getenv('LC_CTYPE', '') or 'UTF-8' in os.getenv('LANG', '') or 'UTF-8' in os.getenv('LC_ALL', '')
 ```

Reply:
Could also do:

```
return any('UTF-8' in os.getenv(var, '') for var in ['LC_CTYPE', 'LANG', 'LC_ALL'])
```

as long as it works I don't mind too much. And it just checks these three env vars for UTF-8, I don't care after that point.